python/pylibcudf/pylibcudf/libcudf/interop.pxd

```diff
@@ -134,7 +134,7 @@ cdef extern from *:
     template <typename ViewType>
     ArrowArray* to_arrow_host_raw(
       ViewType const& obj,
-      rmm::cuda_stream_view stream       = cudf::get_default_stream(),
```

Comment:
Just FYI I also see a default, default (is there an echo in here?) stream being set on `to_arrow_device_raw`. Luckily it shouldn't be applicable for cudf_polars but eventually `to_arrow_device_raw` should have the same treatment too